src/screens/Feed/Feed.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
You don't need to import React from 'react' since react 16.x

Reply:
Done